catalog/users/iii-pactdev.json

```
@@ -1,8 +1,8 @@
 {
   "id": "iii-pactdev",
   "kind": "solutionprovider",
-  "name": "Digital Carbon Dioxide Reduction Cloud (DCDRC)",
+  "name": "Institute for Information Industry(III)",
```

I wonder whether a space is missing:
```suggestion
  "name": "Institute for Information Industry (III)",
```

---

Thank you for your reminder.